### PR TITLE
Fix the admin job parameter when ADD job.

### PR DESCRIPTION
**Pull request by Shylock-Hg**
Follow https://github.com/vesoft-inc/nebula-common/pull/273

### CODE REVIEW

File: src/validator/AdminJobValidator.cpp [hunk 1]
```diff
@@ -15,7 +15,23 @@ Status AdminJobValidator::validateImpl() {
 }
 
 Status AdminJobValidator::toPlan() {
-    auto *doNode = SubmitJob::make(qctx_, nullptr, sentence_->getType(), sentence_->getParas());
+    meta::cpp2::AdminCmd cmd = meta::cpp2::AdminCmd::UNKNOWN;
```
Comment (darionyaphet) [inline src/validator/AdminJobValidator.cpp:18]: `cmd` must be initial ?

File: src/executor/admin/SubmitJobExecutor.cpp [hunk 2]
```diff
@@ -17,23 +17,9 @@ folly::Future<Status> SubmitJobExecutor::execute() {
...
-            cmd = meta::cpp2::AdminCmd::COMPACT;
-        } else if (params.front() == "flush") {
-            cmd = meta::cpp2::AdminCmd::FLUSH;
-        } else {
-            DLOG(FATAL) << "Unknown job command " << params.front();
-            return Status::Error("Unknown job command %s", params.front().c_str());
-        }
-    }
-    return qctx()->getMetaClient()->submitJob(jobOp, cmd, sjNode->params())
+    return qctx()->getMetaClient()->submitJob(sjNode->jobOp(), sjNode->cmd(), sjNode->params())
```
Comment (darionyaphet) [inline src/executor/admin/SubmitJobExecutor.cpp:20]: AFAIK `sjNode->params ` don't include the space name ?

File: src/validator/AdminJobValidator.cpp (hunk 1 above)
Comment (Shylock-Hg) [reply src/validator/AdminJobValidator.cpp:18]: Yes or lead to compile error.

File: src/executor/admin/SubmitJobExecutor.cpp (hunk 2 above)
Comment (Shylock-Hg) [reply src/executor/admin/SubmitJobExecutor.cpp:20]: Good catch. Fixed.

**Review by dutor — APPROVED** [review]

**Comment by dutor** [comment]
Please resovle the conflicts

**Comment by laura-ding** [comment]
https://github.com/vesoft-inc/nebula-graph/pull/317 has done it, so close it

**Pull request closed without merging**